lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -3683,3 +3687,40 @@ def test_get_template_and_context(self):
             assert template == 'public_video_share_embed.html'
             assert context['fragment'] == fragment
             assert context['course'] == self.course
+
+class TestCoursewareMFESearchAPI(SharedModuleStoreTestCase):
+    """
+    Testing the Course Search API.
+    """
+
+    def setUp(self):
+        super().setUp()
+
+        self.course = CourseFactory.create()
+
+        self.client = APIClient(enforce_csrf_checks=True)
```

Comment:
probably don't need enforce csrf checks here

Reply:
Guilty. It was the leftover of the code I borrowed.